packit-ci.fmf

```diff
@@ -13,8 +13,8 @@
 
   discover:
     how: fmf
-    url: https://github.com/RedHat-SP-Security/keylime-tests
-    ref: main
+    url: https://github.com/sergio-correia/keylime-tests.git
```

Comment:
It is referencing your private github. It did slip in the commit? If not, maybe is better to add a new PR in the RedHat repo and wait before merging this.

Reply:
Yeah, it was a temp change to run the tests with some different setup (my fork installed `pyasn1`). I removed this change now.